README.md

```diff
@@ -52,9 +52,25 @@ To deploy this solution, perform the follow steps:
     - Service Account Admin
     - Service Usage Admin
 
-1.  To deploy this repository using an online terminal with software and authentication preconfigured, use [Cloud Shell](https://shell.cloud.google.com/?show=ide%2Cterminal).
+1.  Create, train and deploy a custom document classifier
 
-    Alternatively, to deploy this repository using a local terminal:
+    To classify documents, you must [create a custom document classifier in the Google Cloud console](https://cloud.google.com/document-ai/docs/custom-classifier).
```

Comment:
I disagree with moving the custom classifier steps to here. Most of this section is about how to deploy the Terraform resources self-contained in this repo, so it's out of place to require elaborate steps that aren't part of the repo.

(Same argument for the step about the domain)

I suggest we make this subsectionection only for deploying the Terraform. How about restructuring like...
1. Deployment Guide
    1. Deploy the terraform
    2. Configure the custom classifier (this content)
    3. Enable the web application (steps about the domain name ownership, and additional steps about DNS and OAuth config that currently aren't documented)